lib/glimesh_web/live/chat_live/index.ex

```diff
@@ -146,18 +139,7 @@ defmodule GlimeshWeb.ChatLive.Index do
 
   @impl true
   def handle_info({:user_timedout, bad_user}, socket) do
-    # Must tell the JS function to re-assign the timestamps since a timeout seems to re-render the entire DOM.
-    show_timestamps =
-      if socket.assigns.user,
-        do: Accounts.get_user_preference!(socket.assigns.user).show_timestamps,
-        else: false
-
-    {:noreply,
-     socket
-     |> push_event("remove_timed_out_user_messages", %{bad_user_id: bad_user.id})
-     |> push_event("update_previous_messages_with_timestamp_state", %{
-       show_timestamps: show_timestamps
-     })}
+    {:noreply, push_event(socket, "remove_timed_out_user_messages", %{bad_user_id: bad_user.id})}
```

Comment:
The deleted code is no longer required since timing out does not re-render the DOM.